video/out/filter_kernels.h

```diff
@@ -34,7 +34,7 @@ struct filter_kernel {
     bool polar;         // whether or not the filter uses polar coordinates
     // The following values are set by mp_init_filter() at runtime.
     int size;           // number of coefficients (may depend on radius)
-    double inv_scale;   // scale factor (<1.0 is upscale, >1.0 downscale)
+    double filter_scale;   // scale factor (<1.0 is upscale, >1.0 downscale)
```

Comment:
Comment here also sort of doesn't fit the new name; `filter_scale` is more like `how much wider than normal to stretch the filter`.